data/names.txt

```diff
@@ -3518,6 +3448,166 @@ phrase "pirate"
 		"Fate"
 		"Boys"
 
+phrase "pirate"
+	phrase
+		"historical male pirates"
+		"historical female pirates"
+		"historical pirate ships"
+		"historical ghost ships"
+		"fictional male pirates"
+		"fictional female pirates"
+		"fictional pirate ships"
+		"fictional ghost ships"
+
+phrase "historical male pirates"
+	word
+		"Blackbeard"
+		"Edward Teach"
+		"William Kidd"
+		"Long Ben"
+		"Henry Morgan"
+		"Francois l'Olonnais"
+		"Francis Drake"
+		"Cheung Po Tsai"
+		"Bartholomew Roberts"
+		"Benjamin Hornigold"
+		"Calico Jack"
+		"Emanuel Wynn"
+		"Christopher Moody"
+		"Stede Bonnet"
+		"Paulsgrave Williams"
+		"Dragut"
+		"Thomas Tew"
+		"Henry Every"
+		"Philip Lyne"
+		"Kanhoji Angre"
+
+phrase "historical female pirates"
+	word
+		"Lady Killigrew"
+		"Ching Shih"
+		"Anne Bonny"
+		"Mary Read"
+		"Johanna Hård"
```

Comment:
The `å` character and any other accented/non-standard to English characters (I've also spotted a use of `ø`) won't display in game without #4123.

Reply:
The non-ASCII names in this PR are:

 - [Johanna Hård](https://en.wikipedia.org/wiki/Johanna_Hård)
 - [København](https://en.wikipedia.org/wiki/København_(ship))
 - [Bêlit](https://en.wikipedia.org/wiki/Bêlit_(Robert_E._Howard))

My preference is to not merge this PR until #4123 is resolved (rather than ASCII-izing the names to "Johanna Hard:, etc), since I think the 30th Century in the game should reflect a variety of cultures, and that we've finally converted from ASCII to Unicode.

Reply:
Perhaps remove them and put them in a follow-up PR, then?